### DISCUSSION TITLE
5.0.0.3

### DISCUSSION THREAD

**Discussion by electronstudio** [electronstudio/raylib-python-cffi#134] [Announcements]
* `pyray` functions optimised to be almost as fast as `raylib` functions.  **Please report any breakage!**
* Fixed some red underlines in type hints.  Improved some error messages.
* Python 3.13 support.
* Raspberry Pi binaries (tested on Pi4).
* MacOS builds are now MacOS 12 x86 and Macos 14 ARM because CI doesn't support older versions.  You'll have to compile from source if you are on older MacOS.

<hr /><em>This discussion was created from the release <a href='https://github.com/electronstudio/raylib-python-cffi/releases/tag/v5.0.0.3'>5.0.0.3</a>.</em>

**Comment by Hammer2900** [comment]
everything is faster now, thanks for the release.
